### PR TITLE
testing: Add kokoro build on windows 2019

### PR DESCRIPTION
**Pull request by tmatsuo**
---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/2921)

### CODE REVIEW

File: ci/kokoro/windows/continuous-win-2019.cfg [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+# Copyright 2018 Google LLC
```
Comment (coryan) [inline ci/kokoro/windows/continuous-win-2019.cfg]: nit: copyright year is 2019?

File: ci/kokoro/windows/continuous-win-2019.cfg [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+action {
+  define_artifacts {
+    regex: "**/win_minidumps/*.dmp"
+  }
+}
+
+env_vars {
+  key: "BUILD_CACHE"
+  value: "gs://cloud-cpp-kokoro-results/build-artifacts/vcpkg-installed-win-2019.zip"
```
Comment (coryan) [inline ci/kokoro/windows/continuous-win-2019.cfg]: Do we need a similar change for the `-presubmit` build?

File: ci/kokoro/windows/continuous-win-2019.cfg (hunk 1 above)
Comment (tmatsuo) [reply ci/kokoro/windows/continuous-win-2019.cfg]: Done

File: ci/kokoro/windows/continuous-win-2019.cfg (hunk 2 above)
Comment (tmatsuo) [reply ci/kokoro/windows/continuous-win-2019.cfg]: Added presubmit config too

**Pull request merged**